### PR TITLE
Added USD profit calculation for current value button

### PR DESCRIPTION
**Pull request by sydekumf**

### CODE REVIEW

File: btb_manager_telegram/buttons.py [hunk 1]
```diff
@@ -89,14 +138,13 @@ def current_value():
                     f"\nLast update: `{last_update.strftime('%H:%M:%S %d/%m/%Y')}`\n\n"
                     f"*Current coin {current_coin}:*\n"
                     f"\t\- Balance: `{format_float(balance)}` *{current_coin}*\n"
-                    f"\t\- Current coin exchange rate: `{format_float(usd_price)}` *USD*/*{current_coin}*\n"
+                    f"\t\- Exchange rate purchased: `{format_float(buy_price / alt_amount)}` *{bridge}*/*{current_coin}* \n"
+                    f"\t\- Exchange rate now:           `{format_float(usd_price)}` *USD*/*{current_coin}*\n"
```
Comment (lorcalhost) [inline btb_manager_telegram/buttons.py:142]: Why is there a big blank space on this line? Formatting doesn't look good this way.

Comment (sydekumf) [reply btb_manager_telegram/buttons.py:142]: The only reason is that the output is proper aligned like this:
```
 - Exchange rate purchased: 9.437 USDT/QTUM 
 - Exchange rate now:       9.44 USD/QTUM
```

I know maybe there is a more elegant way to do this, please let me know :-) Python is not my main language :-)

Comment (DmytroLitvinov) [reply btb_manager_telegram/buttons.py:142]: We are planning to integrate i18n into the bot and for another language, it can be broken.

Comment (sydekumf) [reply btb_manager_telegram/buttons.py]: Okay, I removed them :-)

Comment (sydekumf) [reply btb_manager_telegram/buttons.py]: What else is needed to merge this PR? :-)

Comment (lorcalhost) [reply btb_manager_telegram/buttons.py]: @sydekumf I was waiting for testing it a bit more myself but so far everything seems ok

**Pull request merged**